Core/src/org/sleuthkit/autopsy/casemodule/NewCaseVisualPanel2.java

```diff
@@ -61,75 +67,61 @@ public String getName() {
-            .addGroup(layout.createSequentialGroup()
-                .addContainerGap()
-                .addComponent(optionalLabel)
-                .addPreferredGap(javax.swing.LayoutStyle.ComponentPlacement.UNRELATED)
-                .addGroup(layout.createParallelGroup(javax.swing.GroupLayout.Alignment.BASELINE)
-                    .addComponent(caseNumberLabel)
-                    .addComponent(caseNumberTextField, javax.swing.GroupLayout.PREFERRED_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.PREFERRED_SIZE))
-                .addGap(18, 18, 18)
-                .addGroup(layout.createParallelGroup(javax.swing.GroupLayout.Alignment.BASELINE)
-                    .addComponent(examinerLabel)
-                    .addComponent(examinerTextField, javax.swing.GroupLayout.PREFERRED_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.PREFERRED_SIZE))
-                .addContainerGap(javax.swing.GroupLayout.DEFAULT_SIZE, Short.MAX_VALUE))
-        );
+        setLayout(new java.awt.BorderLayout());
     }// </editor-fold>//GEN-END:initComponents
+
     // Variables declaration - do not modify//GEN-BEGIN:variables
-    private javax.swing.JLabel caseNumberLabel;
-    private javax.swing.JTextField caseNumberTextField;
-    private javax.swing.JLabel examinerLabel;
-    private javax.swing.JTextField examinerTextField;
-    private javax.swing.JLabel optionalLabel;
     // End of variables declaration//GEN-END:variables
-
     String getCaseNumber() {
-        return caseNumberTextField.getText();
+        return propertiesPanel.getCaseNumber();
+    }
+
+    String getExaminerName() {
+        return propertiesPanel.getExaminerName();
+    }
+
+    String getExaminerPhone() {
+        return propertiesPanel.getExaminerPhone();
+    }
+
+    String getExaminerEmail() {
+        return propertiesPanel.getExaminerEmail();
+    }
+
+    String getCaseNotes() {
+        return propertiesPanel.getCaseNotes();
+    }
+
+    String getOrganization() {
+        return propertiesPanel.getOrganization();
+    }
+
+    void setCaseNumber(String caseNumber) {
+        propertiesPanel.setCaseNumberField(caseNumber);
+    }
+
+    void setExaminerName(String examinerName) {
+        propertiesPanel.setExaminerNameField(examinerName);
+    }
+
+    void setExaminerPhone(String examinerPhone) {
+        propertiesPanel.setExaminerPhoneField(examinerPhone);
+    }
+
+    void setExaminerEmail(String examinerEmail) {
+        propertiesPanel.setExaminerEmailField(examinerEmail);
+    }
+
+    void setCaseNotes(String caseNotes) {
+        propertiesPanel.setCaseNotesField(caseNotes);
+    }
+
+    void setOrganization(String organizationName) {
+        propertiesPanel.setCurrentlySelectedOrganization(organizationName);
     }
 
-    String getExaminer() {
-        return examinerTextField.getText();
+    void refreshCentralRepoFields() {
+        propertiesPanel.setUpCentralRepoFields();
```

Comment:
I think this name (across both classes) is a bit misleading now, given that we are storing several of the values gathered in the case metadata file as well as the CR. Perhaps refresh/setUpCaseDetailsFields?